jetty-websocket/websocket-jetty-client/src/main/java/org/eclipse/jetty/websocket/client/ClientUpgradeRequest.java

```diff
@@ -50,11 +50,12 @@ public ClientUpgradeRequest()
         this.host = null;
     }
 
+    @Deprecated
```

Comment:
Add javadoc `@deprecated` that explains what to use instead.

Reply:
Should we just deprecate all of the ClientUpgradeRequest.requestURI behaviors too? (as in `.getRequestURI()` should probably be deprecated and always return null too.

Reply:
@joakime it implements these methods from the `UpgradeRequest` interface, and these methods need to be used on other implementations of this interface so they can't be deprecated.